minidec/main.rs

```diff
@@ -59,6 +67,7 @@ fn main() {
         //println!("Local Variable info: {:?}", rfn.locals);
         let mut writer: IRWriter = Default::default();
         writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ff);
+        writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ffm);
```

Comment:
Oops. Didn't pay attention to the test cases. They are not building:

`src/analysis/interproc/interproc.rs:100:20: 100:27 error: the trait bound 'std::io::Stdout: std::fmt::Write' is not satisfied [E0277]
src/analysis/interproc/interproc.rs:100             writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut io::stdout());
                                                           ^~~~~~~
src/analysis/interproc/interproc.rs:100:20: 100:27 help: run 'rustc --explain E0277' to see a detailed explanation
src/frontend/ssaconstructor.rs:595:16: 595:23 error: the trait bound 'std::io::Stdout: std::fmt::Write' is not satisfied [E0277]
src/frontend/ssaconstructor.rs:595         writer.emit_il(Some("main".to_owned()), &ssa, &mut io::stdout());
                                                  ^~~~~~~
src/frontend/ssaconstructor.rs:595:16: 595:23 help: run 'rustc --explain E0277' to see a detailed explanation
src/frontend/ssaconstructor.rs:606:16: 606:23 error: the trait bound 'std::io::Stdout: std::fmt::Write' is not satisfied [E0277]
src/frontend/ssaconstructor.rs:606         writer.emit_il(Some("main".to_owned()), &ssa, &mut io::stdout());
                                                  ^~~~~~~